src/contrib/cluster/Akka.DistributedData/VersionVector.cs

```diff
@@ -257,7 +257,7 @@ public SingleVersionVector(UniqueAddress node, long version)
 
         public override bool IsEmpty => false;
         public override int Count => 1;
-        public override IEnumerator<KeyValuePair<UniqueAddress, long>> VersionEnumerator => new Enumerator(Node, Version);
```

Comment:
Is this a breaking API change or is this a subtype of an `internal` type above?

Reply:
Yeah, both of these are breaking changes.

I thought about doing this in https://github.com/akkadotnet/akka.net/pull/4990 - but I decided to avoid the breaking API change by exposing an internal API that should be used for this.

I don't think the risks are that high of this breaking change having a significant negative impact as DData is either primarily used directly or inside of cluster sharding and these APIs aren't called directly... but it's a judgement call.